site2/docs/reference-pulsar-admin.md

```diff
@@ -2369,20 +2348,279 @@ Remove the message TTL for a topic.
 
 Usage
 ```bash
-$ pulsar-admin topics remove-message-ttl tenant/namespace/topic
-=======
+$ pulsar-admin topics remove-message-ttl tenant/namespace/topic 
+```
+
+Options 
+|Flag|Description|Default|
 |---|---|---|
 |`--enable`, `-e`|Enable message deduplication on the specified topic.|false|
 |`--disable`, `-d`|Disable message deduplication on the specified topic.|false|
 
+### `get-deduplication`
+Get a deduplication policy for a topic.
+
+Usage
+```bash
+$ pulsar-admin topics get-deduplication tenant/namespace/topic
+```
+
+### `set-deduplication`
+Set a deduplication policy for a topic.
+
+Usage
+```bash
+$ pulsar-admin topics set-deduplication tenant/namespace/topic options
```

Comment:
Do we now support tenant-level policies?  Seems not merged yet.

https://github.com/apache/pulsar/wiki/PIP-51%3A-Tenant-policy-support